hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/OmSnapshotLocalDataManager.java

```diff
@@ -57,6 +74,13 @@ public void computeAndSetChecksum(Yaml yaml, OmSnapshotLocalData data) throws IO
         data.computeAndSetChecksum(yaml);
       }
     };
+    this.versionNodeMap = new HashMap<>();
+    init();
+  }
+
+  @VisibleForTesting
+  Map<UUID, SnapshotVersionsMeta> getVersionNodeMap() {
+    return versionNodeMap;
```

Comment:
should we return a unmodifiable copy here


```suggestion
    return Collections.unmodifiableMap(new HashMap<>(versionNodeMap));
```